python_modules/libraries/dagster-dbt/dagster_dbt/cli/app.py

```diff
@@ -161,10 +161,8 @@ def project_scaffold_command(
         Syntax(
             code="\n".join(
                 [
-                    f"cd '{dbt_project_dir}' \\",
-                    "  && dbt parse --target-path target \\",
-                    f"  && cd '{dagster_project_dir}' \\",
-                    "  && dagster dev",
+                    f"cd '{dagster_project_dir}' \\",
```

Comment:
Could potentially remove this `cd` if we switch into the newly created directory in this CLI. Would be pretty clean to just display one command.

Reply:
I think it's best how it is. I think cd-ing within a command that does other things is unexpected and makes it harder to script against.